src/utils.ts

```diff
@@ -277,18 +277,22 @@ export function isAlreadySentToChain(id: string): boolean {
 //////////////////////////////////////////////////////////////////////////////////////////
 // limiting delegation number
 

```

Comment:
Since you exported the function please provide documentation as jsDoc comment

Reply:
We dont need to export it. It was done for testing, reverted it